src/xdocs/config_imports.xml

```diff
@@ -1396,7 +1396,7 @@ import java.util.stream.IntStream;
           <tr>
             <td>groups</td>
             <td>
-              list of imports groups (every group identified either by a
+              list of <b>regular</b> import groups (every group identified either by a
```

Comment:
it is better to not invent new terminology and use JDK terminology - https://docs.oracle.com/javase/specs/jls/se9/html/jls-7.html#jls-7.5

type-import vs static-import.

Reply:
>it is better to not invent new terminology

I didn't invent it, i took it from the previous comment
```
policy on the relative order between regular imports and static imports
```

Let's use the term `type import`. Without the hyphen.

Reply:
good.

Reply:
done